numpy/testing/_private/utils.py

```diff
@@ -852,7 +856,7 @@ def func_assert_same_pos(x, y, func=isnan, hasval='nan'):
         raise ValueError(msg)
 
 
-def assert_array_equal(x, y, err_msg='', verbose=True):
+def assert_array_equal(x, y, err_msg='', verbose=True, strict=False):
```

Comment:
```suggestion
def assert_array_equal(x, y, err_msg='', verbose=True, *, strict=False):
```
Same comment as above